core/application/impl/app_config_impl.cpp

```diff
@@ -241,6 +244,7 @@ namespace kagome::application {
     po::options_description additional_desc("Additional options");
     additional_desc.add_options()
         ("single_finalizing_node,f", "if this is the only finalizing node")
+        ("already_synchronized,s", "if need to consider sinchronized")
```

Comment:
```suggestion
        ("already_synchronized,s", "if need to consider synchronized")
```
Also, tbh the comment isn't very clear.